.gitignore

```diff
@@ -7,3 +7,4 @@ report.txt
 report.json
 table_cache_dev_*
 .DS_Store
+.env
```

Comment:
Please set up your editor to all end lines (especially the last line) with a newline.  It's just less hassle with git that way.

I can fix it for this particular instance, but it would be useful to fix your config in general.  Thanks!